chat-components/src/components/notificationpane/interfaces/common/INotificationPaneInternal.ts

```diff
@@ -0,0 +1,71 @@
+import { IStyle } from "@fluentui/react";
+import { ICommandButtonControlProps } from "../../../common/interfaces/ICommandButtonControlProps";
+import { IImageControlProps } from "../../../common/interfaces/IImageControlProps";
+import { INotificationPaneComponentOverrides } from "./INotificationPaneComponentOverrides";
+
+/**
+ * This interface will act as a generic object that will have the common properties and also the properties for each scenario.
+ * Stateful will populate this interface based on the scenario type and send it to NotificationPane component view.
+ */
+export interface INotificationPaneInternal {
```

Comment:
Is this interface only used in chat-widget only then? If so shouldn't we add it to chat-widget folder?

Reply:
This interface is also consumed in chat-components within `NotificationPane.tsx`.